src/languages/php.js

```diff
@@ -313,7 +317,7 @@ export default function(hljs) {
           regex.concat(WHITESPACE, "+"),
           // to prevent built ins from being confused as the class constructor call
           regex.concat("(?!", normalizeKeywords(BUILT_INS).join("\\b|"), "\\b)"),
-          /\\?\w+/,
+          regex.concat("\\\\?", IDENT_RE),
           regex.concat(WHITESPACE, "*\\("),
```

Comment:
```suggestion
          regex.concat(/\\?/, IDENT_RE),
          regex.concat(WHITESPACE, "*", /\(/),
```

Not sure about that top one OTTOMH since I'm not sure what you're trying to match...

Prefer regex always over strings.

Reply:
Whatever, it'll probably all get squashed in the end...